### DISCUSSION TITLE
Welcome to Obsidian Discussions!

### DISCUSSION THREAD

**Discussion by dHIM24** [kalushko26/Obsidian#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OlegPozhenskij** [comment]
Нou have a very cool knowledge base, great material, it's impossible to break away :) It would be interesting to see what you can add about Proxy here:
![0O5bU_k8n9c](https://github.com/kalushko26/Obsidian/assets/57754088/bcc0a17f-5769-42be-9fc5-d303172fc909)
